extensions/google-chrome/package.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "google-chrome-go-to-tab",
+  "title": "Google Chrome Go to Tab",
+  "description": "Go to an opened Google Chrome tab.",
+  "icon": "google_chrome-icon.png",
+  "author": "Codely",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "google-chrome-go-to-tab",
+      "title": "Go to tab",
```

I feel like going to a tab is the action vs. you search a tab. I would suggest to rename the command to `Search Tabs` because you could also simply copy the URL.

```suggestion
      "title": "Search Tabs",
```